packages/devtools_app/lib/src/service/resolved_uri_manager.dart

```diff
@@ -0,0 +1,51 @@
+import '../shared/globals.dart';
+import 'vm_service_wrapper.dart';
+
+/// Manager for handling package Uri lookup and caching.
+class ResolvedUriManager {
+  Map<String, String?>? _resolvedUrlMap;
```

Comment:
add dartdoc explaining that this map maps resolved uris to their resolved package uris.